docs/docs/shards/types.md

```diff
@@ -631,12 +631,14 @@ Its also known as map, data dictionary, or associative array. An example of a **
 
 ## Wire
 
-Type **Wire** is a sequence of shards (i.e., a sequence of [**ShardRef**](#shardref) or [**Shard**](#shardref) types).
+Type **Wire** represents a wire being passed as data. 
```

Comment:
Where do we explain what a wire is? Maybe link to it?

Reply:
Hmm, we also need to a link to Shards explanation for type `ShardsRef` then I guess.

Reply:
Added primer-links for `shard` and `wire` (primer content is WIP, but links are known).